client/package.json

```diff
@@ -33,6 +39,7 @@
     "clipboard-polyfill": "^2.8.1",
     "connected-react-router": "^6.5.0",
     "core-js": "^3",
+    "crypto-browserify": "^3.12.0",
```

Comment:
this package, along with `stream-browserify`, are required by storybook. `stream` and `crypto` are node std-lib packages for which webpack 4 automatically provided polyfills. Webpack 5 doesn't include those polyfills, hence the need to get a browserify'd version.